docs/docs/configuration/hardware_acceleration.md

```diff
@@ -62,6 +64,19 @@ services:
               capabilities: [gpu]
 ```
 
+#### Docker Run CLI
+
+```bash
+docker run -d \
+  --name frigate \
+  ...
+  --gpus=all \
+  --runtime=nvidia \
```

Comment:
I don't think you need the `--runtime` specified if you have the updated nvidia-container-runtime installed.  I think that flag was for the older version of the docker runtime?
 
See docker documentation for [Access an NVIDIA GPU](https://docs.docker.com/config/containers/resource_constraints/#gpu) and [GPU access with Compose](https://docs.docker.com/compose/gpu-support/)

Reply:
Do you know a concise way to explain that? For example I run in Unraid and it's still required

Wondering if a comment is enough to say it may be unnecessary?

Reply:
That's interesting.  I'm not sure how Unraid is setup for the nvidia runtime.  Maybe we just need an Unraid-specific note? Is that in any of the Unraid documentation we can reference?

Reply:
actually, I remember now that the unraid nvidia driver plugin was updated recently as well as the docker dependency in Unraid 6.11.5. I just tried without it and it worked as expected so probably fine to just remove that part for now